### DISCUSSION TITLE
🚀 Feature Request: Member Roles Tags - Multiple Roles

### DISCUSSION THREAD

**Discussion by npuglisi87** [ExpressionEngine/ExpressionEngine#3372] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**
I would like to allow for multiple member roles to be tagged in a single `{exp:member:has_role role_id="6"}` tag. 

**Describe the solution you'd like**
I would assume using semi-typical EE language that we could just delimit the role numbers, for example role_id="1|3|6" and that should show for members who have either/all 1, 3, 6. 

**Describe alternatives you've considered**
EEHarbor's Member Module, but I'd really prefer to keep things native to EE. 

**Teachability, Documentation, Adoption, Migration Strategy**
Small update (I'm happy to do so) to the EE Docs page https://docs.expressionengine.com/latest/member/member-roles-tags.html#expmemberhas_role

**Comment by intoeetive** [comment]
@npuglisi87 thank you for this request!

It looked familiar to me, and indeed we have an older issue for it (which hasn't been converted to discussion) : https://github.com/ExpressionEngine/ExpressionEngine/issues/2749

And we have code for this PRed: https://github.com/ExpressionEngine/ExpressionEngine/pull/2817 (can't say when it will be released just yet though)

With all of this, closing topic as duplicate
